app/workers/web_push_notification_worker.rb

```diff
@@ -9,7 +9,7 @@ def perform(session_activation_id, notification_id)
     session_activation = SessionActivation.find(session_activation_id)
     notification = Notification.find(notification_id)
 
-    return if session_activation.nil? || session_activation.web_push_subscription.nil? || notification.nil?
+    return if session_activation.nil? || notification.nil?
```

Comment:
if i understand garg's feedback correctly this should just be

`return if session_activation.web_push_subscription.nil?`